static/src/stylesheets/module/commercial/_adverts.scss

```diff
@@ -241,6 +335,11 @@
             display: none;
         }
     }
+
+    &:not([open]) .inline-minus,
```

Comment:
Then I need javascript to toggle the classes, but this moots the intent of the `details`.